openwisp_users/tests/test_models.py

```diff
@@ -61,6 +61,18 @@ def test_organization_repr(self):
         org = self._create_org(name='org1', is_active=False)
         self.assertIn('disabled', str(org))
 
+    def test_organization_owner_bad_organization(self):
+        user = self._create_user(username='user1', email='abc@example.com')
+        org1 = self._create_org(name='org1')
+        org2 = self._create_org(name='org2')
+        org_user = self._create_org_user(organization=org1, user=user)
+        org_owner = self._create_org_owner()
+        org_owner.organization = org2
+        org_owner.organization_user = org_user
+        with self.assertRaises(ValidationError):
+            org_owner.full_clean()
+            org_owner.save()
```

Comment:
I think the error would be saved at "org_owner.full_clean()", right?, so please remove `org_owner.save()` from the assertRasises scope.

Why? Because we want only the line that can raise the error and not anyother line, so that if there is any other error in the `org_owner.save()` line, it's not hidden from us! :smile:

Reply:
Yes, but it seems to be like that in all the other tests, why it is so then?

Reply:
I found only one and I removed it: https://github.com/openwisp/openwisp-users/commit/4f64f7cd76987cbeff7115aae7e3187823a84ce2

Do you see other instances of this?

@atb00ker is right.